src/offchainapi/protocol_messages.py

```diff
@@ -190,6 +199,9 @@ def __init__(self):
         self.status = None
         self.error = None
 
+    def __repr__(self):
+        return f"Response <cid:{self.cid} status:{self.status} error:{self.error}>"
+
```

Comment:
Printing a request is a more elaborate business, since it needs to print the payment (need object). I will wait for the use case first to see what we need there?